app.js

```diff
@@ -1,11 +1,82 @@
 /* Imports */
 
 /* Get DOM Elements */
+const cityArticle = document.getElementById('city-article');
+
+const nameInput = document.getElementById('name-input');
+const nameDisplay = document.getElementById('name-display');
+const climateSelect = document.getElementById('climate-select');
+const climateDisplay = document.getElementById('climate-image');
+const architectureSelect = document.getElementById('architecture-select');
+const architectureDisplay = document.getElementById('architecture-image');
+const attractionInput = document.getElementById('attraction-input');
+const addButton = document.getElementById('add-button');
+const attractionList = document.getElementById('attraction-list');
 
 /* State */
+let city = {
+    name: 'Cool Town',
+    climate: 'desert',
+    architecture: 'modern',
+    attractions: [],
+};
+
+/* Helper Functions */
+function pageLoad() {
+    displayControls();
+    displayCity();
+    displayAttractions();
+}
 
 /* Events */
+nameInput.addEventListener('input', () => {
+    city.name = nameInput.value;
+    pageLoad();
+});
+
+climateSelect.addEventListener('change', () => {
+    city.climate = climateSelect.value;
+    pageLoad();
+});
+
+architectureSelect.addEventListener('change', () => {
+    city.architecture = architectureSelect.value;
+    pageLoad();
+});
+
+addButton.addEventListener('click', () => {
+    if (attractionInput.value !== '') {
+        city.attractions.push(attractionInput.value);
+        displayAttractions();
+        attractionInput.value = '';
+    }
+});
```

Comment:
so on line 48, since you only want to run that code when there is something in the input, you can check for ```if (attractionInput.value)```. An empty string is what's know as a falsy value — it's not equal to false (```'' !== false```) but it evaluates to false in conditional checks. So if ```attractionInput.value === ''```, then ```attractionInput.value``` evaluates to false